src/local/base.py

```diff
@@ -0,0 +1,134 @@
+import asyncio
+from contextlib import suppress
+import enum
+import glob
+from logging import getLogger
+import os
+import subprocess
+from typing import Iterable, Optional
+import webbrowser
+from attr import dataclass
+from abc import ABC, abstractmethod
+
+from galaxy.api.types import LocalGame
+
+from .shared import load_vdf
```

Comment:
the imports could use a bit of ordering (if you're using vscode, just run the "organize imports" command from the palette). this also applies to the rest of the files